src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -0,0 +1,92 @@
+import { XXXL } from "@zendeskgarden/react-typography";
+import type { ServiceCatalogItem } from "./data-types/ServiceCatalogItem";
+import styled from "styled-components";
+import { Button } from "@zendeskgarden/react-buttons";
+import { useState } from "react";
+import ChevronUp from "@zendeskgarden/svg-icons/src/16/chevron-up-fill.svg";
+import ChevronDown from "@zendeskgarden/svg-icons/src/16/chevron-down-fill.svg";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+import { useTranslation } from "react-i18next";
+
+const ItemTitle = styled(XXXL)`
+  font-weight: ${(props) => props.theme.fontWeights.semibold};
+`;
+
+const CollapsibleDescription = styled.div<{ expanded: boolean }>`
+  font-size: ${(props) => props.theme.fontSizes.md};
+  text-align: left;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: ${(props) => (props.expanded ? "none" : 3)};
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100%;
+  margin-top: ${(props) => props.theme.space.md};
+`;
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 120px;
+
+  @media (max-width: 768px) {
+    flex-direction: column;
+  }
+`;
```

Comment:
Are we sure we want this behavior for mobile? Just flipping the flex-direction and keeping the 120px gap? From the design, it seems that on mobile the button is a sticky footer, but I am not sure.

I also don't like we are setting the gap to an hardcoded 120px value, but if that is the value we don't have alternatives

Reply:
I removed the 120px value and changed it to `space.xl` and also added this value to the right and left margin property of the columns.

Reply:
I changed a bit style for mobile but I will work more on that. I see that we have some global css properties for footer and container and this sticky button doesn't look like the one on the design.